src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -616,6 +620,45 @@ interface IParameterNbiotSim {
     pin: types.TUint16
 }
 
+/**
+ * Idle channel
+ */
+interface IParameterIdleChannel {
+}
+
+/**
+ * Power channel settings
+ */
+interface IParameterPowerChannel {
```

Comment:
is it ok to be empty?

Reply:
I added the IParameterEmpty interface as an analog to the IEmptyCommandParameters interface and used it to replace the empty IParameterIdleChannel and IParameterPowerChannel interfaces.